packages/@poupe-css/README.md

```
@@ -542,6 +544,179 @@ const cssProperties = Object.entries(styleObject).map(
+// Result: '.test1, .test2'
+
+// Use custom aliases
+const customAliases = { 'custom': '@media (min-width: 1200px)' };
+processCSSSelectors('custom', { aliases: customAliases });
+// Result: '@media (min-width: 1200px), @media (min-width: 1200px) *'
+
+// Mixed selectors and aliases
+processCSSSelectors(['.test', 'mobile'], { addStarVariants: false });
+// Result: ['.test', '@media (max-width: 768px)']
+```
+#### `processCSSSelectors(selectors: string | string[], options?: ProcessCSSSelectorOptions): string | string[] | undefined`
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)